packages/gafl-webapp-service/src/__mocks__/test-utils-system.js

```diff
@@ -10,6 +10,8 @@ import mockConcessions from './data/concessions'
 import mockDefraCountries from './data/defra-country'
 import { pricingDetail } from '../processors/pricing-summary'
 
+process.env.SESSION_COOKIE_PASSWORD = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'
```

Comment:
So that this is a bit clearer, you should do `("A").repeat(32)`

Reply:
done

Reply:
Re-reviewed and this has been actioned